bgpd/bgp_labelpool.c

```diff
@@ -1555,3 +1678,77 @@ void bgp_lp_vty_init(void)
 	install_element(ENABLE_NODE, &clear_labelpool_perf_test_cmd);
 #endif /* BGP_LABELPOOL_ENABLE_TESTS */
 }
+
+DEFINE_MTYPE_STATIC(BGPD, LABEL_PER_NEXTHOP_CACHE,
+		    "BGP Label Per Nexthop entry");
+
+int bgp_label_per_nexthop_cache_compare(
```

Comment:
MPLS label shouldn't be compared?

Reply:
I have a list of nexthops that will be populated by unique MPLS values.
The context helps me finding out if there is an already available context for that nexthop.

Reply:
ok

Reply:
done